app/controllers/scientist_experiments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ScientistExperimentsController < ApplicationController
+  def destroy
+    @scientist = Scientist.find(params[:scientist_id])
+    @experiments = @scientist.experiments
+    @experiment = @experiments.find(params[:id])
+    @scientist_experiment = @scientist.scientist_experiments.find_by(scientist_id: @scientist.id, experiment_id: @experiment.id)
+
+    @scientist_experiment.destroy
+
+    redirect_to scientist_path(@scientist)
+  end
+end
```

Comment:
Good fix! Takes the long way around but gets there eventually. Here's a shortcut:

```ruby
@scientist = Scientist.find(params[:scientist_id])
@experiment = @experiments.find(params[:id])

ScientistExperiment.find_by(scientist_id: @scientist.id, experiment_id: @experiment.id)
```